what-is-mtls.md

```diff
@@ -0,0 +1,191 @@
+# What is MTLS?
+You’ve probably heard about TLS. It is a security protocol that sits between the transport and application layers (in the TCP/IP model) and essentially encrypts application-layer traffic. This encryption provides a guarantee to the client that:
```

Comment:
Let's dig a bit deeper into security guarantees and how exactly TLS achieves those.

Given our security-minded audience we have to be more precise in our statements about TLS. For example, encryption is not providing authentication  https://www.linkedin.com/pulse/encryption-authentication-mark-percival/.

Can you expand this with more technical level detail about encryption, authentication, certificate structure and exchange on the wire with guarantees it provides?